tests/dist/threads/test_openmp.cpp

```diff
@@ -3,12 +3,14 @@
 #include "fixtures.h"
 
 #include <faabric/scheduler/Scheduler.h>
+#include <faabric/util/string_tools.h>
+
+#define PI_FUNCTION "pi_faasm"
```

Comment:
Very minor, but it feels a bit odd that this is the only function with its name `#define`-ed and also that it follows a different format than MPI. I would have expected it to be called `omp_pi`.